### ISSUE TITLE
Check whether there are missing integration tests for RadioButton's label

### ISSUE THREAD

**Issue by vursen** [vaadin/flow-components#2159]
Currently, the RadioButton's integration tests don't fail with the new accessible `<vaadin-radio-button>` implementation. I suspect that there is the same lack of tests as for `Checkbox` described in https://github.com/vaadin/flow-components/issues/2152.

We need to investigate whether there are some missing integration tests for the RadioButton's label.
